driver/bpf/fillers.h

```diff
@@ -3528,12 +3528,15 @@ FILLER(sys_signaldeliver_e, false)
 	ctx = (struct signal_deliver_args *)data->ctx;
 #ifdef BPF_SUPPORTS_RAW_TRACEPOINTS
 	struct siginfo *info = (struct siginfo *)ctx->info;
-
 	sig = ctx->sig;
-	if (sig == SIGKILL) {
+
+	if (info == SEND_SIG_NOINFO || info == SEND_SIG_PRIV || info == SEND_SIG_FORCED) {
```

Comment:
I missed SEND_SIG_FORCED in my change. I think it should be in both the regular+bpf drivers, but both places need to add a version check. It's been removed in later kernel versions according to https://lore.kernel.org/patchwork/patch/981479/.

Have you tried building with a recent kernel? If we have compilation problems and add version checks, let's also update the regular driver code.

Reply:
Ooh, good catch, thanks! Definitely doesn't build on a 5.2 kernel!

It's just a macro, so I think a simple #ifdef might be more straightforward than a version check. I'll update both the kmod and the bpf driver accordingly!